src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedParameterSymbol.cs

```diff
@@ -153,7 +153,12 @@ internal override bool IsCallerMemberName
             get { return _underlyingParameter.IsCallerMemberName; }
         }
 
-        public override string GetDocumentationCommentXml(CultureInfo preferredCulture = null, bool expandIncludes = false, CancellationToken cancellationToken = default(CancellationToken))
+        internal override AttributeAnnotations FlowAnalysisAnnotations
```

Comment:
>internal override AttributeAnnotations FlowAnalysisAnnotations [](start = 8, length = 62)

In general, in symbols only leaf classes should provide implementation and each leaf class should be covered by targeted tests. This implementation is fine for the prototype, but, if we decide to go with this approach, we should follow up. For example, I will not be surprised if retargeting symbols would need special handling, etc. Please add a prototype comment.